avro/README.md

```diff
@@ -111,6 +111,36 @@ byte[] avroData = mapper.writer(schema)
 
 and that's about it, for now.
 
+## Java Time Support
+Serialization and deserialization support for limited set of `java.time` classes to Avro with [logical type](http://avro.apache.org/docs/current/spec.html#Logical+Types) is provided by `AvroJavaTimeModule`.
+
+```java
+AvroMapper mapper = AvroMapper.builder()
+    .addModule(new AvroJavaTimeModule())
+    .build();
+```
+ 
+#### Note
+Please note that time zone information is at serialization. Serialized values represent point in time, 
```

Comment:
Should "is at serialization" instead be "is not included at serialization" (or something like that)?